app/main.ipynb

```diff
@@ -35,8 +35,18 @@
   },
```

Comment:
Standard library imports, then third-party library imports and local imports the last